integration-test/utils/datafactory/schemas/mentorship.schema.ts

```diff
@@ -0,0 +1,38 @@
+import { pageSchema } from './page.schema';
+import { pageSectionSchema } from './pagesection.schema';
+import { feedbackSectionSchema } from './feedback.schema';
+
+export const mentorshipSchema = {
+  $ref: '#/definitions/mentorshipSchema',
+  definitions: {
+    mentorshipSchema: {
+      type: 'object',
+      properties: {
+        id: {
+          type: 'string',
+          minLength: 1,
+        },
+        page: {
+          page: { ...pageSchema.definitions.pageSchema },
```

Comment:
Why do you duplicate `page` property twice? I thought you should provide reference to pageSchema right after the first `page`